zip-1337.md

```diff
@@ -0,0 +1,156 @@
+
+- Published a quarterly tech roadmap reports and financial reports, detailing spending levels/burn rate and cash/ZEC on hand
+
+- (if beginning of calendar year) Published a yearly review of organization performance, along the lines of the [Zcash Foundation's "State of the Foundation" report](https://www.zfnd.org/blog/foundation-in-2019/)
+
+For the Zcash Foundation, the trust MUST further require:
+
+- No board member may have an interest in the ECC (current board members with an interest would need to divest of their ECC holdings prior to the beginning of this dev fund)
+
+- Excluding money restricted for ZF Grants, the Foundation's total assets must stay below $100mm (if its assets ever exceeded this amount from a disbursement, the trust could direct the funds toward an additional restricted ZF Grants donation) 
+
+Additionally, for the ECC, the trust MUST validate the following before each disbursement:
+
+- (if the beginning of fiscal year) The ECC published yearly audited financial statements at the same level of detail as a public company (to mirror the Foundation's Form 990 requirement as 501(c\)(3))
+
+- No outside investment was received while they are obligatory recipients of this dev fund
+
+- No portion of the dev fund went to dividends, profit-sharing, or share/equity buybacks while they are obligatory recipients of this dev fund
+
+- No portion of the dev fund went into speculative investments in other companies while they are obligatory recipients of this dev fund
+
+- No dilution of ECC's equity except in the case of options/RSUs for new/existing employees while they are obligatory recipients of this dev fund
+
+- There's no change-of-control (majority control changes) at the ECC while they are obligatory recipients of this dev fund
+
+The ECC MUST share necessary information with the trust to ascertain no violations of the above, but the information itself (i.e. cap table and detailed financials) SHOULD remain private between the ECC and the trustees unless there is a violation that is not cured.
+
+### The shielded adoption metric
+
+I applaud Eran Tromer's suggestion for tying some portion of the dev fund to shielded adoption. It's a great tool for adding a protocol-verified accountability metric and I hope it's considered in many of the other proposals as well. My particular adaptation of it provides for some floor of funding for both the ECC and Zcash Foundation, while providing a variable payout based on shielded uptake (scaled to 80% as "fully shielded" to incorporate the possibility of lost value in transparent balances and protect against a single entity holding veto power against shielded uptake). Rather than reward the remaining ZEC to miners — which could cause perverse behavior where miners might censor shielded transactions — the trust would hold the remaining ZEC in escrow, to be paid out to both organizations in lump sum upon reaching a "fully shielded" world during the next quarterly disbursement.
```

Comment:
> Rather than reward the remaining ZEC to miners — which could cause perverse behavior where miners might censor shielded transactions — the trust would hold the remaining ZEC in escrow, to be paid out to both organizations in lump sum upon reaching a "fully shielded" world during the next quarterly disbursement.

What happens if Zcash never reaches this "fully shielded" threshold, or not within any reasonable amount of time? Can a stipulation be added along the lines of "if 'fully shielded' status is not reached within five years, the ZEC held by the trust shall be burned by sending to [insert burn address here]"?

Reply:
I'm not personally a fan of burning, and to be honest I'm more convinced that the shielded [metric itself is problematic](https://forum.zcashcommunity.com/t/a-grand-compromise-synthesis-zip-proposal/34812/58), unfortunately. I will be modifying the proposal to remove the metric entirely...though it will still be a top priority of the Foundation regardless of the proposal selected.